superset/utils/core.py

```diff
@@ -283,7 +283,7 @@ def __init__(self, *args, **kwargs):
     # pylint: disable=E0202
     def default(self, o):
         try:
-            vals = {k: v for k, v in o.__dict__.items() if k != "_sa_instance_state"}
+            vals = {k: v for k, v in o.__dict__.items() if k != "_sa_instance_state" and k != 'password'}
```

Comment:
a cleaner way of doing this would be ...`if k not in ("_sa_instance_state", "password")`

Reply:
Right, I'll correct it